sample-mpp-module/build.gradle

```diff
@@ -8,20 +8,20 @@ android {
 }
 
 kotlin {
-    def configureFrameworks = { target ->
-        target.binaries {
-            framework {
-                baseName = "shared"
-                export project(':reaktive')
+    if (System.getenv().keySet().contains("XCODE_VERSION_MAJOR")) {
```

Comment:
Add a comment here that envvar is available only when the task is run from XCode as pre-built step script.

Reply:
Done.